test/e2e/tests/dapp-interactions/provider-api.spec.js

```diff
@@ -9,47 +9,6 @@ const {
 const FixtureBuilder = require('../../fixture-builder');
 
 describe('MetaMask', function () {
-  it('provider should inform dapp when switching networks', async function () {
```

Comment:
I suppose we could add this back with the step to toggle off the setting, like we did with the `wallet_switchEthereumChain` e2e test